doc/examples/cloud-config-run-cmds.txt

```diff
@@ -21,5 +21,5 @@ runcmd:
  # Note: Don't write files to /tmp from cloud-init use /run/somedir instead.
  # Early boot environments can race systemd-tmpfiles-clean LP: #1707222.
  - mkdir /run/mydir
- - [ wget, "http://slashdot.org", -O, /run/mydir/index.html ]
+ - [ curl, "http://slashdot.org", -O, /run/mydir/index.html ]
```

Comment:
These are not valid curl commands. We should update them to commands that can run without error if we want to land this.